server/src/lib.rs

```diff
@@ -211,9 +214,56 @@ where
   Ok(res)
 }
 
+pub fn captcha_espeak_wav_base64(captcha: &str) -> Result<String, LemmyError> {
+  let mut built_text = String::new();
+
+  // Building proper speech text for espeak
+  for mut c in captcha.chars() {
+    let new_str = if c.is_alphabetic() {
+      if c.is_lowercase() {
+        c.make_ascii_uppercase();
+        format!("lower case {} ... ", c)
+      } else {
+        c.make_ascii_uppercase();
+        format!("capital {} ... ", c)
```

Comment:
This will need localisation.

Reply:
Lets just hold off on that until I can get the captcha maintainers to allow custom text, then we can just use numbers instead.... or just all lower case so we don't have to deal with it.

Reply:
Okay that also works.